vision/super_resolution/super_resolution_cnn_2016/README.md

```diff
@@ -0,0 +1,57 @@
+# SuperResolution
```

Comment:
The folder name for these files should be sub_pixel_cnn_2016.
"super_resolution" is already part of the parent folder name